### PR TITLE
Registering v1.2 components

### PR DESCRIPTION
**Pull request by pkedy**
Registering new components added for v1.2

### CODE REVIEW

File: cmd/daprd/main.go [hunk 1]
```diff
@@ -218,39 +223,42 @@ func main() {
...
+				return rabbitmq.NewRabbitMQ(logContrib)
+			}),
+			pubsub_loader.New("redis", func() pubs.PubSub {
+				return pubsub_redis.NewRedisStreams(logContrib)
+			}),
+			pubsub_loader.New("snssqs", func() pubs.PubSub {
+				return pubsub_snssqs.NewSnsSqs(logContrib)
+			}),
 		),
 		runtime.WithNameResolutions(
```
Comment (halspang) [inline cmd/daprd/main.go:263]: Do we need this one?

https://github.com/dapr/components-contrib/pull/743

**Review by halspang — COMMENTED** [review]
Think I might have found a missing one that was introduced after 1.1.

File: cmd/daprd/main.go (hunk 1 above)
Comment (yaron2) [reply cmd/daprd/main.go:263]: Good call out, lets leave this out until we get some clarification on the use case.

https://github.com/dapr/docs/pull/1310#issuecomment-842815489

**Review by yaron2 — APPROVED** [review]

File: cmd/daprd/main.go (hunk 1 above)
Comment (NepPure) [reply cmd/daprd/main.go:263]: Did we forget  https://github.com/dapr/components-contrib/pull/743 ?

**Pull request merged**